test/include/seqan3/test/tmp_filename.hpp

```diff
@@ -93,22 +93,22 @@ class tmp_filename
         auto path_str = tmp_base_dir.string();  // Copy the underlying path to get access to the raw char *.
         if (char * f = mkdtemp(path_str.data()); f != nullptr)  // mkdtemp replaces XXXXXXXX in a safe and unique way.
         {
-            file_path = f;
-            file_path /= std::filesystem::path{f_name};
+            directory_path = f;
+            file_path = directory_path / std::filesystem::path{f_name};
             return;
         }
         throw std::filesystem::filesystem_error("Could not create temporary directory with mkdtemp!",
-                                           tmp_base_dir,
-                                           std::make_error_code(std::errc::bad_file_descriptor));
+                                                tmp_base_dir,
+                                                std::make_error_code(std::errc::bad_file_descriptor));
     }
 
-    /*!\brief Destructs the temporary file path.
-     * Removes the temporary directory and all it's subdirectories and files contained.
+    /*!\brief Destructs the temporary directory path.
+     * Removes the temporary directory and all its subdirectories and files contained.
```

Comment:
```suggestion
     *
     * \details
     *
     * Removes the temporary directory and all its subdirectories and files contained.
```

Reply:
test/include docs are not build anyway 🤷‍♂

Reply:
yes, that would be true. But why then using brief anyway? So either following the style guide 100% or not at all 😏

Reply:
Just saying :)